assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx

```diff
@@ -150,7 +150,7 @@ const ShippingRatesControl = ( {
 	// Prepare props to pass to the ExperimentalOrderShippingPackages slot fill.
 	// We need to pluck out receiveCart.
 	// eslint-disable-next-line no-unused-vars
-	const { extensions, ...cart } = useStoreCart();
```

Comment:
This was already removed (according to the comment) so it seems like [it slipped in by accident on a recent refactor](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4135). Great catch!